db/migrate/20180129133617_create_work_schedules.rb

```diff
@@ -0,0 +1,19 @@
+class CreateWorkSchedules < ActiveRecord::Migration[5.1]
+  def change
+    create_table :work_schedules do |t|
+      t.string :label
+      t.string :human_readable_label
+      t.timestamps
+    end
+
+    create_table :application_draft_work_schedules do |t|
+      t.belongs_to :application_draft, index: true
+      t.belongs_to :work_schedule, index: true
+      t.timestamps
+    end
+
+    timestamp = Time.now.strftime("%Y-%m-%d %H:%M:%S")
+    connection.execute("INSERT INTO work_schedules (label, human_readable_label, created_at, updated_at) VALUES ('full_time', 'Full-time (40 hours per week)', '#{timestamp}', '#{timestamp}');")
```

Comment:
Having this code inside the `change` method makes the migration no longer reversible....

You could in theory wrap it in something like this:
```ruby
reversible do |dir|
      dir.up do
        # code
```
_(written outta my head, not tested)_